srcpkgs/libsearpc/patches/01-glib-include-kludge.patch

```diff
@@ -0,0 +1,20 @@
+  Seafile sources sometimes include searpc-client.h an extern "C" block,
```

Comment:
not that understandable.Furthermore, the patch content make me think the opposite ?

Reply:
Oops. Typo. I'll rewrite, somewhat more clearly I hope.